app/src/ui/history/merge-call-to-action.tsx

```diff
@@ -11,6 +11,7 @@ interface IMergeCallToActionProps {
   readonly dispatcher: Dispatcher
   readonly currentBranch: Branch
   readonly formState: ICompareBranch
+  readonly onMerge: () => void
```

Comment:
:memo: to indicate this callback fires _after_ the merge is completed (so `onMerged`?)